types/types.ts

```diff
@@ -171,3 +171,67 @@ export interface ToastOptions {
     | "bottom-center"
     | "bottom-right";
 }
+
+// Annotated Collections Types
+export interface AnnotatedCollection {
+  id: string;
+  name: string;
+  description?: string;
+  collection_type: "incident" | "wildlife_sighting" | "research_plot";
+  status: "active" | "archived" | "draft";
+  created_by: string;
+  created_at: string;
+  updated_at: string;
+  is_active: boolean;
```

Comment:
Is this used to track whether the incident is "activated" on the front end, by an end user? I reckon not, right? In which case, I'd remove.